services/core-web/src/constants/styles.js

```diff
@@ -10,3 +9,28 @@ export const COLOR = {
   dangerButtonRed: "#BC2929",
   backgroundWhite: "#ffffff",
 };
+
+// Styling types for various document/application status codes
```

Comment:
I feel like this is the wrong location for these types. The purpose of this file, IMO, is to have a location to change styling attributes in a single location.

Reply:
Discussed with Luke, the issue I had with this is that there shouldn't be business domain logic in the styles file.

Reply:
Chatted and resolved in person.